### PR TITLE
Feature/1971 make gtx key optional

### PR DESCRIPTION
**Pull request by timohuber**

### CODE REVIEW

File: pool/app/pool_common/locales/i18n_de.ml [hunk 1]
```diff
@@ -361,6 +362,9 @@ Wenn Sie die Erinnerungen jetzt manuell auslösen werden über den gewählten Na
     "Standorte, an denen Experimente durchgeführt werden. Jede Session muss \
      eine Location haben."
   | MailingLimit -> "Max. generierte Einladungen pro Mailing."
+  | GtxKeyMissing ->
+    "Es wirde kein GTX Api Key hinterlegt, weshalb keine Textnachrichten \
```
Comment (mabiede) [inline pool/app/pool_common/locales/i18n_de.ml:366]:
```suggestion
    "Es wurde kein GTX Api Key hinterlegt, weshalb keine Textnachrichten \
```

File: pool/app/pool_tenant/event.ml [hunk 2]
```diff
@@ -82,5 +82,5 @@ let handle_event pool : event -> unit Lwt.t = function
     Lwt.return_unit
   | GtxApiKeyUpdated (tenant, gtx_api_key) ->
```
Comment (mabiede) [inline pool/app/pool_tenant/event.ml:83]: Is there an event for deleting the key? How to remove it or is it possible to set the key to an empty value?

File: pool/app/pool_common/locales/i18n_de.ml (hunk 1 above)
Comment (timohuber) [reply pool/app/pool_common/locales/i18n_de.ml:366]: thanks

File: pool/app/pool_tenant/event.ml (hunk 2 above)
Comment (timohuber) [reply pool/app/pool_tenant/event.ml:83]: No currently there is no way to delete it. I added a note to the issue. Not sure if I will be able to do that today

Comment (timohuber) [reply pool/app/pool_tenant/event.ml:84]: Done

**Review by mabiede — APPROVED** [review]

**Pull request merged**